### ISSUE TITLE
Berea: should't the distance ...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#1136]
Notes
======
Berea: should't the distance be in miles, and not on kilometers, in the English version?

Environment
======
Environment Key | Value
:--: | :--:
Version |11.0.0
Operating System | Windows_NT
Platform | win32
Release | 10.0.14393
Architecture | x64

Log History
======
```javascript
2017-03-25 11:6:35 I/ts-dashboard.html:1119:4: Feedback Sent
	https://api.github.com/repos/unfoldingWord-dev/ts-desktop/issues/1135

```

**Comment by ChrisJarka** [comment]
@klappy is there a way to capture the contact information of feedback people so we can respond to a question?

**Comment by cdwhitfield73** [comment]
There is a way to do it, but it has not been assigned as a feature to add.  There is an outstanding issue to someday update how feedback works, but is not included in the scope of version 11.1.  Closing this issue.
